tests/easier68k/core/models/test_list_file_srecord.py

```diff
@@ -4,10 +4,11 @@
 
 import pytest
 import json
+import os.path
 
 from easier68k.core.models.list_file import ListFile
 
-file_path = 'easier68k/core/models/test.S68'
+file_path = os.path.dirname(__file__) + '/test.S68'
```

Comment:
do we know that this will never end with a trailing slash?
Also, we should [use `os.path.join`](https://docs.python.org/3.6/library/os.path.html#os.path.join)

Reply:
> do we know that this will never end with a trailing slash?
wouldn't `os.path.join` resolve that if it does? I mean I might be wrong but it would be nice if it does